src/main.js

```diff
@@ -160,7 +160,7 @@ $(document).ready(() => {
       } else {
         $html.toggleClass(SHOW_CLASS);
         $document.trigger(EVENT.TOGGLE, isSidebarVisible());
-        // Load repo when the page first loaded in the layout mode
+        // Ensure the repo being loaded when openning the sidebar
         if (isSidebarVisible()) tryLoadRepo();
```

Comment:
Does this mean whenever users close/open the sidebar, it will reload? That would be very bad (API limit and performance). Unless I'm mistaken.

Reply:
the repo only is reloaded if there are changes in the repo. Here is the logic for executing the reload.
![screen shot 2018-11-18 at 6 51 27 pm](https://user-images.githubusercontent.com/8633840/48672018-0e5bff80-eb63-11e8-8349-f3bfb0389589.png)